pxr/usd/plugin/usdAbc/alembicReader.cpp

```diff
@@ -95,6 +95,12 @@ TF_DEFINE_ENV_SETTING(
     USD_ABC_XFORM_PRIM_COLLAPSE, true,
     "Collapse Xforms containing a single geometry into a single geom Prim in USD");
 
+#if ALEMBIC_LIBRARY_VERSION >= 10709
+TF_DEFINE_ENV_SETTING(
+    USD_ABC_READ_ARCHIVE_USE_MMAP, false,
+    "Collapse Xforms containing a single geometry into a single geom Prim in USD");
```

Comment:
Can you please update the help string?

Reply:
Sorry to have let this one slip. Fix is pushed.

Reply:
Thanks :)